app/models/event.rb

```diff
@@ -41,6 +41,7 @@ class Event < ActiveRecord::Base
   scope :canceled, -> { where(state: 'canceled') }
   scope :withdrawn, -> { where(state: 'withdrawn') }
   scope :highlighted, -> { where(is_highlight: true) }
+  scope :current, -> { joins(:event_type).where('start_time <= ? AND state = ?', Time.current, 'confirmed').map { |e| e if e.end_time > Time.current }.compact }
```

Comment:
This is no longer valid, because we have pushed new code about schedules. Conferences now can have multiple schedules for admins to try out scheduling scenarios, and there is 1 selected_schedule. You need to choose your event's start_time based on the conference.program.selected_schedule